sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Release History
 ## 5.1.0-beta.4 (Unreleased)
+### New features
+- Added new classes, `StringIndexType`, `RecognizeLinkedEntitiesOptions`.
```

Comment:
with the addition of `StringIndexType` the `Length` property needs to be surfaced again :(

Reply:
So, we need to explore the 'length' property in this release as well?

Reply:
They go hand in hand yeah. It should be a pretty small change as the service already returns it and we just need to expose it.
At the same time, if you ran out of time, create an issue and address in next preview.

Reply:
Also, I think your remarks here does not explain what is the new feature which might not be very helpful to our customers. Please take a look at JS for an example of how to do this: https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/textanalytics/ai-text-analytics/CHANGELOG.md

Reply:
I will add more explaination.